File: v2/csharp/src/main/csharp/io/swagger/Model/Groups.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace io.swagger.Model {
  public class Groups {
    

    
    public List<Group> Groups { get; set; }

    

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class Groups {\n");
      
      sb.Append("  Groups: ").Append(Groups).Append("\n");
      
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  
  
}

File: v2/java/src/main/java/vestorly/client/model/Groups.java
package vestorly.client.model;

import java.util.*;
import vestorly.client.model.Group;

import com.wordnik.swagger.annotations.*;
import com.fasterxml.jackson.annotation.JsonProperty;


@ApiModel(description = "")
public class Groups  {
  
  private List<Group> groups = new ArrayList<Group>() ;

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("groups")
  public List<Group> getGroups() {
    return groups;
  }
  public void setGroups(List<Group> groups) {
    this.groups = groups;
  }

  

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class Groups {\n");
    
    sb.append("  groups: ").append(groups).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}
